lnwallet/mock.go

```diff
@@ -197,9 +197,9 @@ func (w *mockWalletController) ListUnspentWitness(int32, int32,
 
 // ListTransactionDetails currently returns dummy values.
 func (w *mockWalletController) ListTransactionDetails(int32, int32,
-	string) ([]*TransactionDetail, error) {
+	string, uint, int) ([]*TransactionDetail, uint64, uint64, error) {
 
-	return nil, nil
+	return nil, 0, 0, nil
```

Comment:
These changes need to be in the previous commit to make that commit compile on its own.
A nice helper for making sure every commit compiles is:
```
git rebase -i origin/master --exec "make unit pkg=... case=_NONE_"
```

Which basically goes through each commit and compiles all unit tests (but doesn't run them), which causes all code to be compiled as well.

Reply:
I squashed the last 2 commits and now they all compile successfully.

Nice helper 👍